packages/common/client-utils/src/generateNames.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import generateStupidName from "sillyname";
```

Comment:
Please make sure that this code can be tree shake way in webpack.   Given that `client-utils` is a common package, there is a risk that this will still be included in other packages that does use `getRandomName` function.